server/patches/@ptc-org+nestjs-query-graphql+4.2.0.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/node_modules/@ptc-org/nestjs-query-graphql/src/types/connection/cursor/page-info.type.js b/node_modules/@ptc-org/nestjs-query-graphql/src/types/connection/cursor/page-info.type.js
+index 00d836d..8eef442 100644
+--- a/node_modules/@ptc-org/nestjs-query-graphql/src/types/connection/cursor/page-info.type.js
++++ b/node_modules/@ptc-org/nestjs-query-graphql/src/types/connection/cursor/page-info.type.js
+@@ -39,7 +39,6 @@ const getOrCreatePageInfoType = () => {
+         tslib_1.__metadata("design:type", String)
+     ], PageInfoTypeImpl.prototype, "endCursor", void 0);
+     PageInfoTypeImpl = tslib_1.__decorate([
+-        (0, graphql_1.Directive)('@shareable'),
```

Comment:
For some reason this directive does not exist with my setup which throws exception so I had to do this "patch"... Definitely need to investigate more on this

Reply:
Hmm strange, I'll try to take a look on that on Wednesday